src/theme/Toggle.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { func, string } from 'prop-types';
+import styled from "styled-components"
+
+const Button = styled.button`
+    float: right;
+    border-radius: 30px;
+    cursor: pointer;
+    font-size: 0.5rem;
+    padding: 0.5rem;
+    outline: none;
+    border: 0;
+    margin: 9px;
+    &:focus {
+        outline: none;
+    }
+`;
+
+const Sun = styled.svg`
+    height: 1rem;
+    width: 1rem;
+    transition: all 0.7s linear;
+    align: center;
+    display: block;
+`;
+
+const Moon = styled.svg`
+    height: 1rem;
+    width: 1rem;
+    transition: all 0.7s linear;
+    align: center;
+    display: block;
+`
+
+const Toggle = ({theme, toggleTheme }) => {
+    return (
+        <Button onClick={toggleTheme}>
+            {theme === "light" ?
```

Comment:
Again, this defaults in the wrong direction. Test the theme for equality to "dark", default to "light" if it fails.